spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -47,50 +47,13 @@
           let(:campaign) { '20150528-ef-aprovacaoharvard' }
           let(:medium) { 'socialmedia-fe' }
           let(:channel) { 'Social' }
-          let(:source) { 'facebook.com' }
+          let(:source) { 'Facebook' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.